drivers/rtc/Kconfig

```diff
@@ -51,5 +51,6 @@ source "drivers/rtc/Kconfig.mc146818"
 source "drivers/rtc/Kconfig.sam"
 source "drivers/rtc/Kconfig.stm32"
 source "drivers/rtc/Kconfig.smartbond"
+source "drivers/rtc/Kconfig.rpi_pico"
```

Comment:
this should be in alphabetical order

Reply:
OK. Noting that lines 52 and 53 aren't in order, would you like me to correct that too? There's a bit of an inconsistency in the codebase between "add things chronologically" vs "add things alphabetically".